.azure-pipelines/apiview.yml

```diff
@@ -105,7 +98,6 @@ stages:
 
           - script: |
               python --version
-              pip install $(PythonParserPackagePath) $(ProtocolParserPackagePath)
               pip install virtualenv aiohttp chardet trio
```

Comment:
Do we need these other python packages still?

Reply:
yes, APIview creates virtual env when generating token. So installing in the base python folder get it packaged to avoid deployment level steps. I am planning to convert current APIView deployment to a unified deployment pipeline simialr to check-enforcer and we can move these steps to prep server to it.  Submitted issue [3098](https://github.com/Azure/azure-sdk-tools/issues/3098) to track this change.